index.js

```diff
@@ -1,8 +1,10 @@
 module.exports = {
   extends: ['tslint:latest', 'tslint-config-prettier'],
   rules: {
+    'await-promise': true,
```

Comment:
Are non-native promises used much around SEEK (including through third-party packages)? We have one repo where `[true, "Bluebird"]` was required to get this to behave.